index.js

```diff
@@ -3,33 +3,140 @@ module.exports = {
         node: true,
         es6: true
     },
-    rules: {
-    },
     parser: '@typescript-eslint/parser',
     parserOptions: {
       sourceType: 'module',
       project: './tsconfig.json'
     },
+    extends: [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended-requiring-type-checking",
+        "prettier",
+        "airbnb-base",
```

Comment:
Let's not put this under base rule. You should make new file "react.js" that extends from index.js and adds react specific rules.